rules/php-function-names-933150.data

```diff
@@ -1,44 +1,249 @@
-__halt_compiler
-apache_child_terminate
+##! File autogenerated by util/php-dictionary-gen with: -a 30 -F 90000 -s ../fp-finder/spell.sh
+array_diff_uassoc
+array_diff_ukey
+array_filter
+array_intersect_uassoc
+array_intersect_ukey
+array_key_exists
+array_map
+array_push
+array_reduce
+array_shift
+array_udiff
+array_udiff_assoc
+array_udiff_uassoc
+array_uintersect
+array_uintersect_assoc
+array_uintersect_uassoc
+array_values
 base64_decode
+base64_encode
+bson_decode
+bson_encode
 bzdecompress
+bzopen
 call_user_func
-call_user_func_array
-call_user_method
-call_user_method_array
-convert_uudecode
+class_exists
+convert_uuencode
+curl_exec
+curl_file_create
+curl_init
+debug_backtrace
+dirname
+error_reporting
+escapeshellarg
+escapeshellcmd
+exif_imagetype
+exif_read_data
+exif_tagname
+exif_thumbnail
+fclose
+file_exists
 file_get_contents
-file_put_contents
+finfo_open
+fopen
+fputs
 fsockopen
+ftp_connect
+ftp_get
+ftp_nb_get
+ftp_nb_put
+ftp_put
+function_exists
+fwrite
+get_cfg_var
+get_class
 get_class_methods
 get_class_vars
+get_current_user
 get_defined_constants
 get_defined_functions
 get_defined_vars
+get_meta_tags
+getcwd
+getenv
+getimagesize
+getlastmod
+getmygid
+getmyinode
+getmypid
+getmyuid
+gzcompress
 gzdecode
+gzdeflate
+gzencode
+gzfile
 gzinflate
+gzopen
+gzread
 gzuncompress
-include_once
-invokeargs
-pcntl_exec
-pcntl_fork
+gzwrite
+hash_file
+hash_hmac_file
+hash_update_file
+header_register_callback
+hex2bin
+highlight_file
+html_entity_decode
+htmlentities
+htmlspecialchars
+htmlspecialchars_decode
+idate
+image2wbmp
+imagecreatefromgif
+imagecreatefromjpeg
+imagecreatefrompng
+imagecreatefromwbmp
+imagecreatefromxbm
+imagecreatefromxpm
+imagegd
+imagegd2
+ini_get
+ini_get_all
+ini_set
+intval
+iptcembed
+is_a
+is_array
+is_dir
+is_executable
+is_file
+is_int
+is_null
+is_numeric
+is_object
+is_readable
+is_writable
+is_writeable
+iterator_apply
+json_decode
+json_encode
+mb_ereg
+mb_ereg_match
+mb_ereg_replace
+mb_ereg_replace_callback
+mb_eregi
+mb_eregi_replace
+mb_parse_str
+md5
```

Comment:
**md5**
I think this is dangerous for API that could contains `hash=md5` could lead to FPs
Maybe it's a good idea to move this out, isn't it?